core/metrics-core-service/src/main/antlr4/org/hawkular/metrics/core/service/tags/parser/TagQuery.g4

```diff
@@ -67,6 +69,10 @@ array_operator
   | NOT IN
   ;
 
+existence_operator
+  : NOT
```

Comment:
I think that's correct.
(Now both metrics and alerting are sync-ed with the expression language).
I.e.:
tagA and (not tagB or (tagC in [a, 'bc.*'] and tagD not in [1, 2]))